pages/instructorAssessmentAccess/instructorAssessmentAccess.ejs

```diff
@@ -46,7 +46,7 @@
           </table>
         </div>
         <div class="card-footer">
-          <small>Instructions on how to change the access rules can be found in the <a href="https://prairielearn.readthedocs.io/en/latest/accessControl/" target="_blank">PrairieLearn documentation</a>. Note that changing some of these rules does not affect assessments in progress; to change the time limit for these exams please visit the <a href="instances">Students tab</a>.</small>
+          <small>Instructions on how to change the access rules can be found in the <a href="https://prairielearn.readthedocs.io/en/latest/accessControl/" target="_blank">PrairieLearn documentation</a>. Note that changing time limit rules does not affect assessments in progress; to change the time limit for these exams please visit the <a href="instances">Students tab</a>.</small>
```

Comment:
(Warning! untested suggested change)

Nit: prefer explicit URLs. This is because relative URLs mean that we have to be extremely careful with trailing slashes on URLs. That is, if we are on a current page with `.../access/` versus `.../access` then the current page will display the same, but a relative URL of `instances` will go to either `.../access/instances` (incorrect) or `.../instances` (correct).

```suggestion
          <small>Instructions on how to change the access rules can be found in the <a href="https://prairielearn.readthedocs.io/en/latest/accessControl/" target="_blank">PrairieLearn documentation</a>. Note that changing time limit rules does not affect assessments in progress; to change the time limit for these exams please visit the <a href="<%= urlPrefix %>/assessment/<%= assessment.id %>/instances">Students tab</a>.</small>
```

Reply:
Tested and works.